src/devices/ExplorerHat/Motorization/Motor.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Device.Gpio;
+
+namespace Iot.Device.ExplorerHat.Motorization
+{
+    /// <summary>
+    /// Represent one of the onboard motors
+    /// </summary>
+    public class Motor : IDisposable
+    {
+        private GpioController _controller;
+
+        private DCMotor.DCMotor _innerMotor;
+
+        /// <summary>
+        /// Motor number on Hat
+        /// </summary>
+        public int Number { get; private set; }
+
+        /// <summary>
+        /// GPIO pin to control speed
+        /// </summary>
+        public int SpeedControlPin { get; private set; }
+
+        /// <summary>
+        /// GPIO pin to control direction
+        /// </summary>
+        public int DirectionPin { get; private set; }
+
+        /// <summary>
+        /// Current speed
+        /// </summary>
+        public double Speed
+        {
+            get
+            {
+                return _innerMotor.Speed;
+            }
+            set
+            {
+                double speed;
+
+                if (value > 1)
+                {
+                    speed = 1;
+                }
+                else if (value < -1)
+                {
+                    speed = -1;
+                }
+                else
+                {
+                    speed = value;
+                }
+
+                _innerMotor.Speed = speed;
```

Comment:
you should just forward this to DCMotor and if you think the default behavior is wrong then file issue on DCMotor, they should work consistently the same.